cnd/src/http_api/routes.rs

```diff
@@ -136,15 +136,15 @@ fn make_siren_action(swap_id: LocalSwapId, action_name: &str) -> siren::Action {
 pub struct AliceHanEthereumHalightBitcoinState {
     pub alpha_ledger_state:
         LedgerState<asset::Ether, htlc_location::Ethereum, transaction::Ethereum>,
-    pub beta_ledger_state: halight::State,
+    pub beta_ledger_state: hlnbtc::State,
```

Comment:
```suggestion
    pub beta_ledger_state: halight_btc::State,
```
I do not find `hltnbtc` readable.

Reply:
I agree with @D4nte - we chose `halight` because it is nicely readable and spell-able. @thomaseizinger ?

Reply:
Fair points, `hlnbtc` is not readable and `halight` is nicer.